README.md

```diff
@@ -106,6 +106,25 @@ jobs:
       uses: actions/checkout@v2
 ```
 
+#### `total length of command line and environment variables exceeds limit`
+
+If the error `total length of command line and environment variables exceeds limit` appears, then
+the current environment variables' total size has exceeded the maximum when executing Go Wasm binaries.
+
+To resolve this issue, install `cleanenv` and use it to prefix your command.
+
+For example, if this is the failing line in GitHub Actions:
+```bash
+go test -cover ./...
+```
+The new commands should be the following:
+```bash
+go install github.com/agnivade/wasmbrowsertest/cmd/cleanenv@latest
+cleanenv -remove-prefix GITHUB_ -- go test -cover ./...
+```
+
+The `cleanenv` command above removes all environment variables prefixed with `GITHUB_` before running the command after the `--`.
```

Comment:
Add a second sentence "You can use the flag multiple times to strip other environment variables as well."

Reply:
Fixed in 6a93001